### DISCUSSION TITLE
Attribute selectors specificity

### DISCUSSION THREAD

**Discussion by afzalsayed96** [stitchesjs/stitches#270] [Ideas] [answered]
Branching off from #83 to start a discussion about handling the specificity for attribute selectors.

**Problem Description**

Consider the following stylesheet:

```
button:hover {}
button[disabled] {}
```

Since pseudo selectors and attribute selectors have same [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), one would expect disabled styles take precedence over hover styles. 

However, as we've (synthetically) increased specificity of pseudo selectors in #112, attribute selectors always end up having a lower precedence.

Take a look at this sandbox for repro: https://codesandbox.io/s/hungry-grothendieck-oo5m0?file=/src/App.tsx

**Comment by afzalsayed96** [comment, marked as answer]
I wanted to check what's the roadmap for this issue.
This is currently a blocker for us for completely migrating our codebase from react-jss 😞 
Should we create an issue so it can be tracked as a bug?

cc @peduarte  @hadihallak

**Reply by peduarte** [reply]
Oh yeah thats solved in beta :) https://codesandbox.io/s/pedantic-mclean-yy1rc?file=/src/App.tsx
